lms/static/js/discovery/result_item_view.js

```diff
@@ -0,0 +1,50 @@
+;(function (define) {
+
+define([
+    'jquery',
+    'underscore',
+    'backbone',
+    'gettext',
+    'date'
+], function ($, _, Backbone, gettext, Date) {
+    'use strict';
+
+    function formatDate(date) {
+        return dateUTC(date).toString('MMM dd, yyyy');
```

Comment:
@chrisndodge did some client-side date formatting for notifications, would that be useful here?